.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -24,12 +24,12 @@ contact_links:
   - name: Commercial and community support
```

Comment:
Wondering if we should re-order t his to: Community an commercial support

Reply:
Sure!